Makefile.am

```diff
@@ -18,6 +18,6 @@ FSRC = fsrc
 FTEST = ftest
 endif
 
```

Comment:
Add a section like the one for Fortran. Like this:

```
# Does the user want the Python API?
if BUILD_PYTHON
PSRC = psrc
PTEST = ptest
endif
```

Reply:
Also go into the configure.ac file, find the section for enable-fortran, and copy it, changing fortran to python...